<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanTabPage</class>
 <widget class="QWidget" name="ScanTabPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScanTabPage</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <widget class="QTableWidget" name="FoundTable">
     <property name="font">
      <font>
       <family>Sarasa Mono SC</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>15</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>350</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sarasa Mono SC</family>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="Placehold1">
        <property name="font">
         <font>
          <family>Sarasa UI SC</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="frameBase">
        <property name="font">
         <font>
          <family>Sarasa UI SC</family>
         </font>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="Base10">
           <property name="font">
            <font>
             <family>Sarasa UI SC</family>
            </font>
           </property>
           <property name="text">
            <string>10进制</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="Base16">
           <property name="font">
            <font>
             <family>Sarasa UI SC</family>
            </font>
           </property>
           <property name="text">
            <string>16进制</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="Base2">
           <property name="font">
            <font>
             <family>Sarasa UI SC</family>
            </font>
           </property>
           <property name="text">
            <string>2进制</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="ExcludeCheckbox">
           <property name="font">
            <font>
             <family>Sarasa UI SC</family>
            </font>
           </property>
           <property name="statusTip">
            <string>扫描时排除指定的值</string>
           </property>
           <property name="text">
            <string>排除</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ScanValueLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sarasa UI SC</family>
         </font>
        </property>
        <property name="text">
         <string>数值:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="ScanValueA">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Sarasa UI SC</family>
            </font>
           </property>
           <property name="maxLength">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ScanValueB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Sarasa UI SC</family>
            </font>
           </property>
           <property name="maxLength">
            <number>999999999</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sarasa UI SC</family>
         </font>
        </property>
        <property name="text">
         <string>数值类型:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="ScanValueType">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sarasa UI SC</family>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ScanMethod">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sarasa UI SC</family>
           </font>
          </property>
          <property name="placeholderText">
           <string>扫描方式</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sarasa UI SC</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QFrame" name="frame_2">
        <property name="font">
         <font>
          <family>Sarasa UI SC</family>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="ScanBtn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Sarasa UI SC</family>
            </font>
           </property>
           <property name="text">
            <string>扫描</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="ScanUndoBtn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Sarasa UI SC</family>
            </font>
           </property>
           <property name="text">
            <string>撤销</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="ScanResetBtn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Sarasa UI SC</family>
            </font>
           </property>
           <property name="text">
            <string>重新开始</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <spacer name="verticalSpacer_3">
        <property name="font">
         <font>
          <family>Sarasa UI SC</family>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QGroupBox" name="FrameMemory">
        <property name="font">
         <font>
          <family>Sarasa UI SC</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="title">
         <string>快速配置</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="MemoryWriteable">
             <property name="font">
              <font>
               <family>Sarasa UI SC</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>可写</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="MemoryExecuteable">
             <property name="font">
              <font>
               <family>Sarasa UI SC</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>可执行</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="MemoryCopyOnWrite">
             <property name="font">
              <font>
               <family>Sarasa UI SC</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>写时拷贝</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="ModuleTable">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Sarasa Mono SC&quot;;</string>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderShowSortIndicator" stdset="0">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
